src/main/java/iris/task/Event.java

```diff
@@ -0,0 +1,60 @@
+package iris.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import iris.exception.DateTimeException;
+
+/**
+ * Represents an event with a start and end date
+ */
+public class Event extends Task {
+    private final LocalDateTime from;
+    private final LocalDateTime to;
+
+    /**
+     * Constructor for Event
+     * @param name description of event
+     * @param from the start date/time of the task in the format: {dd-MM-yyyy HHmm}. Eg. 20-02-2002 2000
+     * @param to the end date/time of the task in the format: {dd-MM-yyyy HHmm}. Eg. 20-02-2002 2000
+     * @throws DateTimeException when there is a problem parsing given dates.
+     */
+    public Event(String name, String from, String to) throws DateTimeException {
+        super(name);
+        try {
+            this.from = LocalDateTime.parse(from.trim(), DateTimeFormatter.ofPattern("dd-MM-yyyy HHmm"));
```

Comment:
Just a suggestion but you could perhaps create another class like `DatetimeParser` to help with parsing of the datetime string!

Reply:
Oh I was thinking of making a method for that (it's kinda messy and repetitive now), but I think a class might be better. Thank you!